### ISSUE TITLE
sharpdx.WPF demo Why not run it?

### ISSUE THREAD

**Issue by shao200** [helix-toolkit/helix-toolkit#162]
I use a subproject of project Examples that come with it, basically could not be run.
Whether the project is sharpdx.Sln, and sharpdx_net40.Sln.
for example:
  public class BillboardTextModel3D : MeshGeometryModel3D
    {
        #region Private Class Data Members

```
    private EffectVectorVariable vViewport;
    private ShaderResourceView billboardTextureView;
    private EffectShaderResourceVariable billboardTextureVariable;
```

Class EffectVectorVariable , ShaderResourceView is Not defined.
These phenomena are basically occurs in every project.Conflicts are either not defined or definition.

**Comment by shao200** [comment]
The previous issue is not resolved, the problem appeared again later.

**Comment by shao200** [comment]
it can not run that in vs 2015 ,pre 3.0.0 bata

**Comment by shao200** [comment]
错误	33	未知的生成错误“未能从程序集“SharpDX, Version=3.0.0.0, Culture=neutral, PublicKeyToken=b4dcf0f35e5521f1”中加载类型“SharpDX.Matrix”。 行 50 位置 14.”	F:\GitHub\helix-toolkit\Source\Examples\WPF.SharpDX\EnvironmentMapDemo\MainWindow.xaml	50	14	EnvironmentMapDemo

**Comment by shao200** [comment]
错误	36	命名空间“clr-namespace:EnvironmentMapDemo”中不存在“MainViewModel”名称。	F:\GitHub\helix-toolkit\Source\Examples\WPF.SharpDX\EnvironmentMapDemo\MainWindow.xaml	14	9	EnvironmentMapDemo

**Comment by shao200** [comment]
错误	25	未能从程序集 \SharpDX.Toolkit.CompilerTask.dll 加载任务“SharpDX.Toolkit.EffectCompilerTask”。未能加载文件或程序集“file:///F:\SharpDX.Toolkit.CompilerTask.dll”或它的某一个依赖项。系统找不到指定的文件。 请确认 <UsingTask> 声明正确，该程序集及其所有依赖项都可用，并且该任务包含实现 Microsoft.Build.Framework.ITask 的公共类。	HelixToolkit.Wpf.SharpDX

**Comment by shao200** [comment]
sharpdx WPF really is too much trouble.

**Comment by SiNeumann** [comment]
I saw this Issue on various installations where the d3dcompiler_xx.dll was missing. According to Microsoft it should be included in Windows 8.1 http://blogs.msdn.com/b/chuckw/archive/2012/05/07/hlsl-fxc-and-d3dcompile.aspx . . For my Application I resolved this issue with LoadLibrary() in the static constructor of the WPF-Control and redistribute the d3dcomplier_xx.dll with my application, this way is suggested by xoofx of the SharpDx-Project https://github.com/sharpdx/SharpDX/issues/683. Since the Visual Studio 2012/13 include the d3dcompiler by default, it could lead to some "works on my machine" issues.
